src/pages/index.js

```diff
@@ -25,28 +36,135 @@ const cardData = {
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   jobSelector: ".profile__description",
+  avatarSelector: ".profile__image",
 });
 
+api.getUserInfo().then((userData) => {
+  //console.log("Setting avatar to:", userData.avatar);
+  //console.log("User Data from API:", userData);
+  userInfo.setUserInfo({
+    name: userData.name,
+    job: userData.about,
+    avatar: userData.avatar,
+  });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**